src/components/HotelCard.jsx

```diff
@@ -39,11 +53,16 @@ function HotelCard({
           </div>
           <div className="tag">
             <img className="icon-dollar" src={dollarIcon} alt="dollar" />
-            {price}
+            {dollars(price)}
           </div>
-          <div>
-            {availabilityFrom}
-            {availabilityTo}
+          <div className="tag">
+            <img
+              className="icon-availability"
+              src={availabilityIcon}
+              alt="availability"
+            />
+            Del {formatUnixToNaturalDate(availabilityFrom)} al{" "}
+            {formatUnixToNaturalDate(availabilityTo)}
```

Comment:
Si quieres evitar el `{" " }` puedes usar template strings, like:

```suggestion
            {`Del ${formatUnixToNaturalDate(availabilityFrom)} al ${formatUnixToNaturalDate(availabilityTo)}`}
```